msal/src/Microsoft.Identity.Client/Platforms/netstandard1.1/PlatformInformation.cs

```diff
@@ -25,16 +25,37 @@
 //
 //------------------------------------------------------------------------------
 
-using System;
-using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Internal;
 
-namespace Microsoft.Identity.Core
+namespace Microsoft.Identity.Client
 {
-    internal class CryptographyHelper
+    internal class PlatformInformation : PlatformInformationBase
     {
-        public static string CreateSha256Hash(string input)
+        public override string GetProductName()
         {
+            return "MSAL.Facade";
```

Comment:
>"MSAL.Facade" [](start = 19, length = 13)

not sure I understand this naming, thoughts?

Reply:
this is only provided as a facade implementation across platforms. I could also have not implemented this and simply put the code invoking the class in a ifdef.

---
In reply to: [161069188](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/941#discussion_r161069188) [](ancestors = 161069188)

Reply:
thanks for explaining.

---
In reply to: [161135225](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/941#discussion_r161135225) [](ancestors = 161135225,161069188)